nbs/01_lhs_parsing.ipynb

```diff
@@ -10,7 +10,7 @@
   },
```

Comment:
<p>did you run <code>nbdev_clean</code> before submitting? this shouldnt show up in published nb </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DeanLight/graph_rewrite/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>I did not - thank you!</p>